Lib/test/test_httpservers.py

```diff
@@ -1536,6 +1541,261 @@ def test_server_test_ipv4(self, _):
+                mock_func.assert_called_once_with(**self.args)
+                self.args['tls_cert'] = None
+                self.args['tls_key'] = None
+                mock_func.reset_mock()
+
+        # Normal: --tls-cert, --tls-key and --tls-password-file
+
+        for tls_cert_option in tls_cert_options:
+            for tls_key_option in tls_key_options:
+                for tls_password_option in tls_password_options:
+                    self.invoke_httpd(tls_cert_option,
+                                      self.tls_cert,
+                                      tls_key_option,
+                                      self.tls_key,
+                                      tls_password_option,
+                                      self.tls_password_file)
+                    self.args['tls_cert'] = self.tls_cert
+                    self.args['tls_key'] = self.tls_key
+                    self.args['tls_password'] = self.tls_password
+                    mock_func.assert_called_once_with(**self.args)
+                    self.args['tls_cert'] = None
+                    self.args['tls_key'] = None
+                    self.args['tls_password'] = None
+                    mock_func.reset_mock()
+
+        # Abnormal: --tls-key without --tls-cert
+
+        for tls_key_option in tls_key_options:
+            for tls_cert_option in tls_cert_options:
+                with self.assertRaises(SystemExit):
+                    self.invoke_httpd(tls_key_option, self.tls_key)
+                    mock_func.reset_mock()
+
+        # Abnormal: --tls-password-file without --tls-cert
+
+        for tls_password_option in tls_password_options:
+            with self.assertRaises(SystemExit):
+                self.invoke_httpd(tls_password_option, self.tls_password_file)
+                mock_func.reset_mock()
+
+        # Abnormal: --tls-password-file cannot be opened
+
+        non_existent_file = 'non_existent_file'
+        for tls_password_option in tls_password_options:
+            for tls_cert_option in tls_cert_options:
+                with self.assertRaises(SystemExit):
+                    self.invoke_httpd(tls_cert_option,
+                                      self.tls_cert,
+                                      tls_password_option,
+                                      non_existent_file)
+
+    @mock.patch('http.server.test')
+    def test_no_arguments(self, mock_func):
+        self.invoke_httpd()
+        mock_func.assert_called_once_with(HandlerClass=self.default_handler,
+                                          ServerClass=self.default_server,
+                                          protocol=self.default_protocol,
+                                          port=self.default_port,
+                                          bind=self.default_bind,
+                                          tls_cert=None,
+                                          tls_key=None,
+                                          tls_password=None)
```

Comment:
```suggestion
        mock_func.assert_called_once_with(**self.args)
```

I think this is 'args' right? if so, I suggest renaming 'self.args' into 'self.default_args'